plugins/providers/virtualbox/provider.rb

```diff
@@ -56,9 +56,6 @@ def machine_id_changed
 
       # Returns the SSH info for accessing the VirtualBox VM.
       def ssh_info
-        # If the VM is not running that we can't possibly SSH into it
```

Comment:
You would need to inspect the VMs status and ensure it's running, paused, or halted I think.